docs/how-to-docs/drone-ci.md

```diff
@@ -444,3 +445,47 @@ OPTIONS:
    --build.commit value    build commit sha
    --build.message value   build commit message
 ```
+### Resource Intensive Builds
+
+As Drone CI v1 builds are now Kubernetes native, builds now do not consume CPU and memory resources based off agents but are rather are run as separate Kubernetes pods within ephemeral namespaces. Therefrore, you can now be explictly declare your pipeline's resource utilisation on a per-step basis (requests, limits, or both). This is particularly useful when executing resource intensive steps as part of your pipeline to ensure its optimal running:
```

Comment:
can you avoid using 'builds' twice in one sentence? and perhaps break that sentence down so it's clearer. also, there's a typo in 'therefore'